main.js

```diff
@@ -33,7 +55,7 @@ function deleteTask(id){
 };
 
 function fetchTasks (){
-    const tasks = JSON.parse(localStorage.getItem('tasks'));
+    const tasks = JSON.parse(localStorage.getItem('tasks')) || [];
```

Comment:
array as alternative value is required because localStorage.getItem would return 'null' by default if no item is found, and app will broke when we operate with 'tasks=null'